### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by BrunoMine**
Some general adjusts and upgrades
Fix bugs:
* Fix nil self.current_node error
* Fix nil current_pos error
* Fix nil def error

Upgrades:
* Add mesh scale setting
* Upgrade eat mode
* Add eat grass sounds
* Modes documentation

### CODE REVIEW

File: creatures/features/general/hit_bottom.lua [hunk 1]
```diff
@@ -137,7 +137,7 @@ end)
 
 -- Check player hit bottom
 creatures.check_player_hit_bottom = function(player, loop)
-	if not player then return end
+	if not player or not player:getpos() then return end
```
Comment (MoNTE48) [inline creatures/features/general/hit_bottom.lua]: player:get_pos()

**Comment by sofar** [comment]
@BrunoMine I gave you commit access. Please help find suitable reviewers.

File: creatures/features/general/hit_bottom.lua (hunk 1 above)
Comment (BrunoMine) [reply creatures/features/general/hit_bottom.lua]: Fixed.

**Comment by BrunoMine** [comment]
@sofar All original reviewers of this project are @lnjX and @DiogoGomes. I'm going to look for new ones, but I can't guarantee anyone will be ready for it now.

**Comment by BrunoMine** [comment]
I am grateful to @MoNTE48 and @AntumDeluge for the contribution.

**Comment by MoNTE48** [comment]
I will consider the possibility of transferring some MC mobs to this engine and, perhaps, I will make some contribution in the future. I'm interested in comparing the performance and usability of the API.
Thank all for contribution and development of an alternative mobs engine.

**Comment by MoNTE48** [comment]
_Maybe offtop, but shouldn't you merge the develop branch to master (which has not been updated for a 2 years!), and start merging some of the changes from this PR?
Now this mod looks dead!_

**Comment by BrunoMine** [comment]
Really. This PR no longer makes any sense.

**Pull request closed without merging**